package.json

```diff
@@ -13,7 +13,7 @@
     "fetch": "github/fetch#^2.0.3",
     "jquery": "jquery/jquery-dist#^3.2.1",
     "promise-polyfill": "taylorhakes/promise-polyfill#^6.0.2",
-    "xterm": "^4.4.0",
+    "xterm": "^4.5.0",
```

Comment:
Please pin this and all xterm addons we're using to the specific versions we're using here. That way we'll avoid this kind of problems in the future.